packages/devtools_app/lib/src/shared/utils.dart

```diff
@@ -44,16 +42,15 @@ void debugLogger(String message) {
 }
 
 double scaleByFontFactor(double original) {
-  return (original * (ideTheme?.fontSizeFactor ?? 1.0)).roundToDouble();
+  return (original * (ideTheme.fontSizeFactor ?? 1.0)).roundToDouble();
```

Comment:
make fontsizefactor non-nullable. To do this you can give IdeTheme.fontSize a default value in the constructor `unscaledDefaultFontSize`

Reply:
done